packages-exp/messaging-exp/src/helpers/register.ts

```diff
@@ -29,27 +29,18 @@ import {
   onSubChange
 } from '../listeners/sw-listeners';
 
+import { GetTokenOptions } from '../interfaces/public-types';
+import { MessagingInternal } from '../interfaces/internal-dependencies';
```

Comment:
I think the CI error is because the source code never imports `@firebase/messaging-interop-types`, which is where the `declare` block that adds `messaging-internal` to `NameServiceMapping` is. I think a good place to import it is here, import the `MessagingInternal` type from that package instead of from `internal-dependencies`, and maybe it can be removed from `internal-dependencies`.

Reply:
got you. You are right. I am able to build now